src/ui/Navbar.fs

```diff
@@ -4,67 +4,125 @@ open Feliz
 open Feliz.Router
 open type Store.UI.NextUI
 open type Feliz.Html
+open Fable.Core.JsInterop
 
 [<ReactComponent>]
 let DefaultNavbar () =
+    let (isMenuOpen, setIsMenuOpen) = React.useState false
+
+    let menuItems = [ "Shop"; "About Us"; "Contact" ]
+
     Navbar [
-        prop.style [ style.backgroundColor "#ffffff" ]
+        prop.custom ("onMenuOpenChange", setIsMenuOpen)
+
+        prop.style [
+            (if isMenuOpen then "#F7F4F0" else "rgba(255,255,255,0)") |> style.backgroundColor
+            style.fontFamily "Figtree"
+            style.fontWeight 400
+            style.float'.left
+            length.px 44 |> style.minHeight
+            (if isMenuOpen then 0.0 else 0.2) |> style.transitionDelaySeconds
+        ]
 
         prop.children [
             NavbarBrand [
                 prop.children [
-                    p [
-                        prop.text "ZORI"
-                        prop.className [ "font-bold"; "text-inherit" ]
+                    img [
+                        prop.src <| import "default" "./assets/Logo.svg"
                         prop.onClick (fun _ -> Router.navigatePath "/")
-                        prop.style [ style.cursor "pointer" ]
+                        prop.style [
+                            length.px 26 |> style.height
+                            length.px 26 |> style.minHeight
+                            style.cursor "pointer"
+                            (if isMenuOpen then 0 else 100) |> style.filter.invert
+                        ]
                     ]
                 ]
             ]
 
             NavbarContent [
-                prop.className [ "sm:flex"; "gap-4" ]
+                prop.className [ "hidden"; "sm:flex"; "gap-4" ]
+                prop.style [ length.vw 5 |> style.gap ]
                 justify "center"
                 prop.children [
+
                     NavbarItem [
                         Link [
-                            prop.text "Products"
-                            color "foreground"
-                            prop.onClick (fun _ -> Router.navigatePath "/products")
-                            prop.style [ style.cursor "pointer" ]
+                            prop.text "Shop"
+                            prop.onClick (fun _ -> Router.navigatePath "/")
```

Comment:
Is this the right path?

Reply:
It's going to be an anchor link for MVP since we only have 1-2 products. So I linked it to homepage as a placeholder.

Reply:
Consider replacing this with a link to where the page _will_ be.

Reply:
The page _is_ homepage.